validator_client/src/doppelganger_service.rs

```diff
@@ -182,7 +182,7 @@ async fn beacon_node_liveness<'a, T: 'static + SlotClock, E: EthSpec>(
                 OfflineOnFailure::Yes,
                 |beacon_node| async move {
                     beacon_node
-                        .post_lighthouse_liveness(validator_indices, previous_epoch)
+                        .post_validator_liveness_epoch(previous_epoch, validator_indices.to_vec())
```

Comment:
Looks like we're creating a `Vec` here unnecessarily. The `validator_indices` passed into this function is actually a `Vec` before it gets shadowed `as_slice` below (so we're extracting a slice from `Vec` and then converting it into a new `Vec`, which isn't needed):
https://github.com/sigp/lighthouse/blob/65d01cae26f18f149b6014defd3b166b88f2b5ac/validator_client/src/doppelganger_service.rs#L160-L166

I think we should be able to remove this now, and keep it as `Vec ` and update the `post_validator_liveness_epoch` to accept a `&Vec` so we don't need to create extra `Vec`s and slices references.